depends/hosts/darwin.mk

```diff
@@ -1,11 +1,18 @@
-OSX_MIN_VERSION=10.12
+OSX_MIN_VERSION=10.14
 OSX_SDK_VERSION=10.14
 OSX_SDK=$(SDK_PATH)/MacOSX$(OSX_SDK_VERSION).sdk
-darwin_CC=clang -target $(host) -mmacosx-version-min=$(OSX_MIN_VERSION) --sysroot $(OSX_SDK)
-darwin_CXX=clang++ -target $(host) -mmacosx-version-min=$(OSX_MIN_VERSION) --sysroot $(OSX_SDK) -stdlib=libc++
 
-darwin_CFLAGS=-pipe
-darwin_CXXFLAGS=$(darwin_CFLAGS)
+clang_prog=$(build_prefix)/bin/clang
+clangxx_prog=$(clang_prog)++
+llvm_config_prog=$(build_prefix)/bin/llvm-config
+
+clang_resource_dir=$(build_prefix)/lib/clang/$(native_cctools_clang_version)
+
+darwin_CC=$(clang_prog) -target $(host)  -isysroot$(OSX_SDK)
+darwin_CXX=$(clangxx_prog) -target $(host) -isysroot$(OSX_SDK) -stdlib=libc++
+
+darwin_CFLAGS=-pipe -std=c11
+darwin_CXXFLAGS=-pipe -std=c++17
```

Comment:
In bitcoin, use `C_STANDARD` and `CXX_STANDARD` like follow:

```
darwin_CFLAGS=-pipe -std=$(C_STANDARD)
darwin_CXXFLAGS=-pipe -std=$(CXX_STANDARD)
```

should we specify the version here?

Reply:
I didn't add new variables as it was used in exactly one place. In Bitcoin these standards are specified in multiple host files.